### PR TITLE
fix syntax wrong while defining a fucntion at line 141

### PR DESCRIPTION
**Pull request by MephistoMMM**
***

Although this function is defined in comment, it may mislead many people ,  like me ... **T _ T**
For this function  is a useful !

### CODE REVIEW

**Comment by mbbill** [comment]
Thanks for the patch but I don't see anything wrong here. The function is supposed to be defined in you vimrc which is why ":s" is used.

**Comment by MephistoMMM** [comment]
But it catch error  when I define undotree_CostomMap() like follow

```vimscript
function g:undotree_CustomMap()
    map <buffer> <c-n> J
    map <buffer> <c-p> K
endfunction
```

vim error info

so I mind it mislead user.

**Comment by mbbill** [comment]
According to line 364, let's use capitalised function name, and '!' is not necessary since it's a user-defined function.
Please update your pr, and then I'll do the merge. Thanks.

**Comment by MephistoMMM** [comment]
Thanks for merging my pr ~(≧▽≦)/~

File: plugin/undotree.vim [hunk 1]
```diff
@@ -138,7 +138,7 @@ endif
 "You can define whatever mapping as you like, this is a hook function which
 "will be called after undotree window initialized.
 "
-"function g:undotree_CustomMap()
+"function Undotree_CustomMap()
```
Comment (mbbill) [inline plugin/undotree.vim]: g:Undotree..

**Comment by MephistoMMM** [comment]
...sorry

**Comment by mbbill** [comment]
no problem.
btw, pls don't keep updating the pr with new patches, you may just merge them into one and do a force push to your pr.

**Comment by MephistoMMM** [comment]
wait , let me try ...

**Comment by MephistoMMM** [comment]
Done, it's amusing

**Comment by mbbill** [comment]
Thanks.

**Pull request merged**